### PR TITLE
all rubocop offenses addressed.

### PR DESCRIPTION
**Pull request by Emmett-42**
All offenses addressed.
Some adjustments made in the .rubocop.yml file to resolve Metrics/MethodLength offense, setting max to 20.

### CODE REVIEW

**Comment by Emmett-42** [comment]
resolved issue #2

File: Gemfile [hunk 1]
```diff
@@ -26,6 +26,8 @@ gem "stimulus-rails"
 
 # Build JSON APIs with ease [https://github.com/rails/jbuilder]
 gem "jbuilder"
+gem 'rubocop'
```
Comment (piratechicken) [inline Gemfile]: Do we need both these? I have a feeling that rubocop is included in rubocop-rails.

**Review by piratechicken — CHANGES_REQUESTED** [review]
One comment in line.

Your PR description mentions `.rubocop.yml` but I can't see it. I'm going to be very annoying and suggest single quotes for strings (unless you're doing interpolation), just because that's the more comon in ruby (and is the rubocop default).

**Review by piratechicken — APPROVED** [review]
Thank you!

**Pull request merged**